api/src/main/java/ca/bc/gov/educ/api/batchgraduation/reader/ArchiveSchoolReportsPartitioner.java

```diff
@@ -48,6 +48,11 @@ public Map<String, ExecutionContext> partition(int gridSize) {
         }
 
         DistributionSummaryDTO summaryDTO = (DistributionSummaryDTO)jobExecution.getExecutionContext().get("distributionSummaryDTO");
+        if(summaryDTO == null) {
```

Comment:
Duplication of code. Is the sumaryDTO not set on the controller when we fire the job? Reference [here](https://github.com/bcgov/EDUC-GRAD-BATCH-GRADUATION-API/blob/develop/alex-GRAD2-2283/api/src/main/java/ca/bc/gov/educ/api/batchgraduation/controller/JobLauncherController.java#L938)

Reply:
Scheduled jobs bypass controllers and run thru job launcher directly. That's why summaryDTO was missing in the execution context

Reply:
Ah ok right.